components/onboarding/CreateAccount.tsx

```diff
@@ -0,0 +1,100 @@
+'use client';
+
+import { useState } from 'react';
+import { redirect } from 'next/navigation';
+import toast from 'react-hot-toast';
+// import { z } from 'zod';
+
+import { createUser } from '@/lib/actions/user.actions';
+import { userSchema } from '@/lib/validations/userSchema';
+
+import Input from '../shared/ui/Input';
+import Button from '../shared/ui/Button';
+import { signIn } from 'next-auth/react';
+
+const CreateAccount = () => {
+  const [data, setData] = useState({
+    name: '',
+    email: '',
+    password: '',
+  });
+  // const [zodErrors, setZodErrors] = useState({});
```

Comment:
I see you were blocking this out. Let's implement them and then get this merged.